.goreleaser.yml

```diff
@@ -21,15 +21,25 @@ builds:
       - arm64
       - arm
       - "386"
+      - ppc64
+      - ppc64le
```

Comment:
Do these systems use systemd?

Reply:
I think they do. I believe Dylan tested this against Alma Linux on PowerPC. We may want to clarify this requirement in the release notes, as we have not implemented the init script yet for legacy systems.